src/python/WMCore/MicroService/Unified/MSManager.py

```diff
@@ -1,21 +1,34 @@
 """
-File       : UnifiedTransferorManager.py
+File       : MSManager.py
 Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
-Description: UnifiedTransferorManager class provides full functionality of the UnifiedTransferor service.
+Description: MSManager class provides full functionality of the MSManager service.
+It provides a REST interface to reqmgr2ms service and should be
```

Comment:
The REST interface is actually provided under the Services package, no?

Reply:
done